app/views/devise/registrations/edit.html.erb

```diff
@@ -1,40 +1,43 @@
-<!--page heading-->
-<header class="pt-5 pb-5">
-  <div class="container">
-    <div class="text-center">
-      <h1 class="section-heading text-uppercase underline">
-        Edit my user settings
-      </h1>
-    </div>
-  </div> <!--container-->
-</header>
+<% if current_user.staff_account %>
+  <%= provide(:header_title, "Edit my user settings")  %>
```

Comment:
I have not used these partials yet so not sure why the header title is in a conditional? Could we just set it as `Edit my user settings` regardless of user?

Reply:
Since dashboard uses a shared layout it utilises a [yield](https://github.com/rubyforgood/pet-rescue/blob/main/app/views/layouts/dashboard.html.erb#L230) for it's different possible titles. The account settings form should provide different header style depending if it is rendered inside dashboard or standalone. To better illustrate what I mean, if I omit the condition the following would happen:

<img width="1224" alt="Screenshot 2023-10-31 at 05 05 33" src="https://github.com/rubyforgood/pet-rescue/assets/31761693/d5efd109-297f-41bf-8120-bb5c58fffc9b">


In the PR description screenshot you see that this condition adds the title to the appropriate spot :)

Reply:
Ahh great, thank you for the explanation @BakiVernes